### PR TITLE
@mzikherman - Pass timezone

### PR DESCRIPTION
**Pull request by broskoski**
This updates moment, deprecates `convert_to_utc` and adds a new option to pass in a timezone string.

See http://momentjs.com/timezone/

### CODE REVIEW

File: schema/fields/date.js [hunk 1]
```diff
@@ -7,16 +8,19 @@ export default {
     format: {
       type: GraphQLString,
     },
+    timezone: {
+      type: GraphQLString,
+    },
     convert_to_utc: {
       type: GraphQLBoolean,
-      defaultValue: true,
+      deprecationReason: 'Use timezone instead',
```
Comment (mzikherman) [inline schema/fields/date.js]: Should this just be removed completely? Doesn't look like it gets used at all anymore.

**Comment by mzikherman** [comment]
❌

File: schema/fields/date.js (hunk 1 above)
Comment (broskoski) [inline schema/fields/date.js]: Yeah, just didn't want to break Force in the meantime

**Comment by broskoski** [comment]
💚

File: schema/fields/date.js (hunk 1 above)
Comment (mzikherman) [inline schema/fields/date.js]: Ah, got it.

**Pull request merged**